homeassistant/components/device_tracker/config_entry.py

```diff
@@ -284,6 +300,10 @@ def state_attributes(self) -> dict[str, StateType]:
         return attr
 
 
+class ScannerEntityDescription(BaseTrackerEntityDescription, frozen_or_thawed=True):
+    """A class that describes tracker entities."""
+
+
```

Comment:
I think we can default the source type of ScannerEntityDescription to ROUTER. We don't intend anything else to be used currently.

Reply:
I've set as default on the `ScannerEntityDescription`

Reply:
> I've set as default on the `ScannerEntityDescription`

This was reverted, as it is now set on the shorthand attribute